src/Rest_API/Shipping/Item_Info.php

```diff
@@ -993,7 +993,8 @@ protected function check_insurance_amount_limit( $backend_data, $order_total ) {
 		} // For EU shipments, validate that insurance does not exceed €5000
 		elseif ( ! $is_non_eu_shipment && 'yes' === $backend_data['insured_shipping'] && $order_total > 5000 ) {
 			throw new \Exception(
-				__( 'Insurance amount for EU shipments cannot exceed €5000. Your total is: ' . $order_total, 'postnl-for-woocommerce' )
+				// Translators: %s is the order total.
+				sprintf( __( 'Insurance amount for EU shipments cannot exceed €5000. Your total is: %s', 'postnl-for-woocommerce' ), wp_kses_post( $order_total ) )
```

Comment:
```suggestion
				sprintf( esc_html__( 'Insurance amount for EU shipments cannot exceed €5000. Your total is: %d', 'postnl-for-woocommerce' ), $order_total )
```
$order_total is a number so we can use %d instead of %s